### PR TITLE
Export misc

### PR DESCRIPTION
**Pull request by benrudolph**
@NoahCarnahan more random stuff. commit message break down changes

cc: @sravfeyn @proteusvacuum

### CODE REVIEW

File: corehq/apps/export/models/new.py [hunk 1]
```diff
@@ -371,9 +371,6 @@ class ExportInstance(BlobMixin, Document):
     # Whether to automatically convert dates to excel dates
     transform_dates = BooleanProperty(default=True)
 
-    # Whether to include duplicates and other error'd forms in export
-    include_errors = BooleanProperty(default=False)
-
```
Comment (proteusvacuum) [inline corehq/apps/export/models/new.py]: I'm assuming `CaseExportInstance` doesn't require the use_errors property?

**Comment by proteusvacuum** [comment]
:+1:

**Comment by sravfeyn** [comment]
Looking good. Build fail looks related.

File: corehq/apps/export/models/new.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/export/models/new.py]: yep

**Comment by benrudolph** [comment]
thanks will fix soon

File: corehq/apps/export/static/export/spec/data/export_instances.js [hunk 2]
```diff
@@ -102,6 +105,7 @@ SampleExportInstances = {
             }],
             "doc_type": "TableConfiguration"
         }],
-        "doc_type": "ExportInstance"
+        "doc_type": "ExportInstance",
+        "is_deidentified": false
```
Comment (dimagimon) [inline corehq/apps/export/static/export/spec/data/export_instances.js]: Missing trailing comma. (comma-dangle)

File: corehq/apps/export/static/export/spec/data/export_instances.js [hunk 3]
```diff
@@ -49,13 +50,15 @@ SampleExportInstances = {
...
```
Comment (dimagimon) [inline corehq/apps/export/static/export/spec/data/export_instances.js]: Missing trailing comma. (comma-dangle)

**Comment by benrudolph** [comment]
✅ (confusing, but the second to last commit passes on tests, just not on lint)

**Comment by benrudolph** [comment]
✅

**Pull request merged**